docs/docs/concepts/structure/_instance_description.mdx

```diff
@@ -1,5 +0,0 @@
-An instance is the top hierarchy in the ZITADEL.
-Within an instance all the default [settings](/docs/concepts/structure/policies), such as branding, login policy, password policy, etc. for the system can be configured.
-One instance normally runs on one domain and has one issuer. (e.g login.customer.com)
-
-One instance can contain multiple [organizations](/docs/concepts/structure/organizations). Which can represent the own company or the customers.
```

Comment:
I think this is also used in another part. i think guides/console

Reply:
I can't find it
![image](https://user-images.githubusercontent.com/12727842/195016217-6d7d8cb3-abba-4af8-adab-040ef31c5b53.png)